src/components/announcements-list/AnnouncementsList.tsx

```diff
@@ -0,0 +1,60 @@
+import { Row } from 'antd';
+import React from 'react';
+import { AnnouncementCard } from '../AnnouncementCard';
+import styled from 'styled-components';
+import axios, { AxiosInstance } from 'axios';
+
+const COLUMNS_PER_ROW = 3
+const NO_ANNOUNCEMENTS_MESSAGE = "There are currently no announcements!"
+const ANNOUNCEMENTS = '/api/v1/announcements/'
+const AuthAxiosInstance: AxiosInstance = axios.create({
+    baseURL: process.env.REACT_APP_API_DOMAIN,
+    timeout: 10000,
+    headers: {
+        'Content-Type': 'application/json',
+    },
+});
+    
+const AnnouncementRow = styled(Row)`
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-bottom: 20px;
+`
+
+export const AnnouncementsList: React.FC = () => {
+    const [announcements, setAnnouncements] = React.useState<Array<any>>([]);
+
+    React.useEffect(() => {
+        (async function () {
+            let announcementsData = await AuthAxiosInstance.get(ANNOUNCEMENTS)
+                .then(response => response.data)
+                .then(data => { return data.announcements as any[] })
+                .catch(() => { return [] as any[] });
+            setAnnouncements(announcementsData);
+        })()
+    }, []);
```

Comment:
We should be loading this data with Redux and storing it there. Please take a look at https://docs.c4cneu.com/guides/frontend-redux/ for a guide and https://github.com/Code-4-Community/speak-for-the-trees-frontend/pull/25 for an example. Happy to pair on this with you and Varun as well :)

Reply:
Will give it a go and let you know if I come across a roadblock 👌